src/app/containers/ATIAnalytics/index.test.jsx

```diff
@@ -228,14 +233,35 @@ describe('ATI Analytics Container', () => {
       amp.default = mockAmp;
 
       render(
-        <ContextWrap platform="amp" pageType={STORY_PAGE} service="news">
+        <ContextWrap platform="amp" pageType={STORY_PAGE} service="mundo">
+          <ATIAnalytics data={styAssetData} />
+        </ContextWrap>,
+      );
+
+      expect(mockAmp.mock.calls[0][0]).toMatchInlineSnapshot(`
+        Object {
+          "pageviewParams": "s=598343&s2=62&p=mundo.story.23263889.page&r=\${screenWidth}x\${screenHeight}x\${screenColorDepth}&re=\${availableScreenWidth}x\${availableScreenHeight}&hl=00-00-00&lng=\${browserLanguage}&x1=[urn%3Abbc%3Acps%3Af776ad93-e486-b14a-b5ea-55955dd0644f]&x2=[amp]&x3=[news-mundo]&x4=[es]&x5=[\${sourceUrl}]&x6=[\${documentReferrer}]&x7=[article]&x8=[simorgh]&x9=[WS%2BSTY%2BTEST%2B-%2BFull%2BHeadline%2B-%2BBBC%2BNews%2BMundo]&x11=[1970-01-01T00%3A00%3A00.000Z]&x12=[1970-01-01T00%3A00%3A00.000Z]&x13=[Life~Fake%2Bnews]&x14=[0239ab33-1cfc-4f5d-babb-a8159711af3e~e7539dc8-5cfb-413a-b4fe-0ad77bc665aa]&x16=[Amuse%20me]&x17=[News]&ref=\${documentReferrer}",
+        }
+      `);
+    });
+
+    it('should call AmpATIAnalytics when platform is Amp and pageType is CSP', () => {
+      const mockAmp = jest.fn().mockReturnValue('amp-return-value');
+      amp.default = mockAmp;
+
+      render(
+        <ContextWrap
+          platform="amp"
+          pageType={CORRESPONDENT_STORY_PAGE}
+          service="mundo"
```

Comment:
should this example maybe be `news`, I don't think there are csp pages for mundo?
```suggestion
          service="news"
```

Reply:
There aren't, but changing it to news was more complicated and meant other fields changed their values and fixture data would need tweaking, so it felt okay to just keep it as mundo to focus the test on the specific element of page type